report/6.Conclusion/Futurework.tex

```diff
@@ -0,0 +1,61 @@
+\section{Future work}\label{sec:futurework}
+Throughout the development of a project, there are often several ideas that are deprioritized for one reason or another.
+This section will describe the ideas that the group had, our initial thoughts, how they might have been implemented and why they were not implemented.
+
+
+\subsection{Predict target location in x frames}
+A \textbf{Should have} requirement from the MosCow analysis in Section~\ref{subsec:requirements}, was the ability to predict the location x frames into the future.
+In the actual implementation this was not done, as a different solution proved better.
```

Comment:
påstand? vi har ikke testet det andet, så ved vi om den er bedre?

Reply:
Har ændret til: "due to time contraints"